### PR TITLE
Code inspector results do not respect 'Jump to ADT first' setting #1892

### PR DESCRIPTION
**Pull request by bigld**
enable jump to ADT and show subobjects (eg Includes) in CI result
Tested with report with include, maybe there is a different behaviour with other objects?

### CODE REVIEW

File: src/zcl_abapgit_branch_overview.clas.abap [hunk 1]
```diff
@@ -8,19 +8,20 @@ CLASS zcl_abapgit_branch_overview DEFINITION
   PUBLIC SECTION.
 
     INTERFACES zif_abapgit_branch_overview .
+    TYPES: ty_commits TYPE STANDARD TABLE OF zif_abapgit_definitions=>ty_commit WITH DEFAULT KEY .
```
Comment (larshp) [inline src/zcl_abapgit_branch_overview.clas.abap]: changes to this include should not be part of this PR?

File: src/objects/zcl_abapgit_objects_super.clas.abap [hunk 2]
```diff
@@ -306,6 +308,14 @@ CLASS zcl_abapgit_objects_super IMPLEMENTATION.
 
         CONCATENATE 'adt://' sy-sysid <lv_uri> INTO lv_adt_link.
 
+        IF i_line_number IS NOT INITIAL.
+          lv_line_number = i_line_number.
+          SHIFT lv_line_number LEFT DELETING LEADING ' 0'.
+          CONDENSE lv_line_number NO-GAPS.
+          CONCATENATE lv_adt_link '/source/main#start=' lv_line_number ',0;end=' lv_line_number ',1 ' INTO lv_adt_link.
```
Comment (larshp) [inline src/objects/zcl_abapgit_objects_super.clas.abap]: did you test with both CLAS and PROG? as far as I remember the links are different for the 2 object types

**Comment by bigld** [comment]
you are right - try to find some method / generic approach

File: src/zcl_abapgit_branch_overview.clas.abap (hunk 1 above)
Comment (bigld) [reply src/zcl_abapgit_branch_overview.clas.abap]: No - messed up my repos...

File: src/objects/zcl_abapgit_objects_super.clas.abap (hunk 2 above)
Comment (bigld) [reply src/objects/zcl_abapgit_objects_super.clas.abap]: Works fine now for all objects

**Comment by bigld** [comment]
Fixed commits 
works fine now for all objetcs also with jump to line

**Comment by bigld** [comment]
#1892

**Comment by bigld** [comment]
removed commented coding
